[2022-11-17T22:47:28.685+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_availible scheduled__2022-11-14T00:00:00+00:00 [queued]>
[2022-11-17T22:47:28.705+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_availible scheduled__2022-11-14T00:00:00+00:00 [queued]>
[2022-11-17T22:47:28.707+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-17T22:47:28.708+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-17T22:47:28.709+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-17T22:47:28.734+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_availible> on 2022-11-14 00:00:00+00:00
[2022-11-17T22:47:28.742+0000] {standard_task_runner.py:55} INFO - Started process 99 to run task
[2022-11-17T22:47:28.753+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_availible', 'scheduled__2022-11-14T00:00:00+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpdzla3eoh']
[2022-11-17T22:47:28.758+0000] {standard_task_runner.py:83} INFO - Job 111: Subtask is_api_availible
[2022-11-17T22:47:28.909+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_availible scheduled__2022-11-14T00:00:00+00:00 [running]> on host bca668ad4dbc
[2022-11-17T22:47:29.076+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_availible
AIRFLOW_CTX_EXECUTION_DATE=2022-11-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-14T00:00:00+00:00
[2022-11-17T22:47:29.079+0000] {http.py:120} INFO - Poking: api/
[2022-11-17T22:47:29.103+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2022-11-17T22:47:29.111+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2022-11-17T22:47:30.125+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2022-11-17T22:47:30.152+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_availible, execution_date=20221114T000000, start_date=20221117T224728, end_date=20221117T224730
[2022-11-17T22:47:30.252+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-17T22:47:30.350+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
